dev-libs/spdlog/spdlog-1.4.2.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils
```

Comment:
One reason for conflict is the port to `cmake.eclass` of older ebuilds, yet this bump is still outdated.